websites/A/Animes Portal/dist/metadata.json

```
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "version": "1.0.0",
+  "author": {
+    "name": "Angelo II",
+    "id": "728512329888825396"
+  },
+  "service": "Animes Portal",
+  "description": {
+    "en": "Anime, manga and Asian culture",
+    "bg": "Аниме, манга и азиатска култура"
+  },
+  "url": "animes-portal.info",
+  "logo": "https://i.imgur.com/EwyBmHJ.png",
+  "thumbnail": "https://i.imgur.com/Nr3XhzA.jpg",
+  "color": "#365AA8",
+  "category": "anime",
+  "iframe": true,
+  "tags": [
+    "anime",
+    "online",
+    "otaku",
+    "animes-portal",
+    "manga",
+    "asia",
+    "japan"
+  ],
+  "settings": [
+    {
+      "id": "showthumb",
+      "title": "Show thumbnails",
+      "icon": "fas fa-image",
+      "value": true
+    },
+    {
+      "id": "showmessaging",
+      "title": "Show my chats",
```

Use something less personal
```suggestion
      "title": "Show chats",
```